jobserv/git_poller.py

```diff
@@ -188,8 +191,16 @@ def _get_repo_changes(refs_cache, url: str, refs: List[str],
                         yield {'GIT_REF': ref, 'GIT_URL': url,
                                'GIT_OLD_SHA': cur, 'GIT_SHA': sha}
 
+def _is_skipped(*messages: str) -> bool:
+    """
+    determines whether commit qualifies as skipped commit
+    :param messages: commit description or commit title
+    :return: if skip flags found in either description or title
+    """
+    return any(commit_skip_pattern.search(msg) for msg in messages)
```

Comment:
this is clean, but really inefficient. Each caller can concatenate `title + body` and pass it as one string to this function. Then you eliminate a loop. The regex is clean but that's also really inefficient when you are just searching for two exact strings. This could just be:
```
def _is_skipped(msg: str) -> bool:
    return '[ci skip]' in msg or '[skip ci]' in msg
```

Reply:
lol - what's funny is I went the regex route for speed.  I keep forgetting how slow `re` is for simple queries.  Thanks for the catch.  The fix should be live now.